Core/Sources/Models/WeatherUpdate.swift

```diff
@@ -126,7 +126,7 @@ public extension WeatherUpdate {
     }
 
     @objc var precipitationPercentage: Double {
-        return Double(todaysForecast["precipProbability"] as? String ?? "") ?? 0
+        return (todaysForecast["precipProbability"] as? NSNumber)?.doubleValue ?? 0
```

Comment:
At least for consistency, I wonder if we should make a similar change (casting via `NSNumber`) for `windSpeed` and `windBearing`?

Reply:
What would be the advantage of doing this?